js/interactive-guides/modules/content-manager.js

```diff
@@ -759,6 +760,22 @@ var contentManager = (function() {
         }
     };
 
+    // May need to change for cases where you have to wait a set time before activating
+    var enableNextInstruction = function(stepName){
+      var stepInstruction = contentManager.getCurrentInstruction(stepName);
+      var nextInstructionIndex = contentManager.getCurrentInstructionIndex(stepName); //this is the instruction that needs to be activated
+
+      if(nextInstruction) {
```

Comment:
should this be nextInstructionIndex?     And doesn't getCurrentInstructionIndex always return something?   So shouldn't it be checking if nextInstructionIndex !== -1?